evap/staff/tests/test_views.py

```diff
@@ -1929,6 +1930,32 @@ def setUpTestData(cls):
         cls.answer = 'should show up'
         baker.make(TextAnswer, contribution=contribution, question=question, answer=cls.answer)
 
+        cls.evaluation2 = baker.make(
+            Evaluation,
+            pk=2,
+            course=baker.make(Course, semester=semester),
+            participants=[student1],
+            voters=[student1, cls.student2],
+            vote_start_datetime=datetime.datetime.now() - datetime.timedelta(days=5),
+            vote_end_date=datetime.date.today() - datetime.timedelta(days=4),
+            can_publish_text_results=True
+        )
```

Comment:
We don't specifically need the pk to be 2 here, right? If so, I'd remove it.

`course=...` could be simplified to `course__semester=semester` (here and above where this was taken from)